doc/release/0.2.rst

```diff
@@ -14,6 +14,7 @@ What's new in ARMI
 #. Use TemporaryDirectoryChanger for executer.run() so dirs are cleaned up during run. (`PR#1219 <https://github.com/terrapower/armi/pull/1219>`_)
 #. New option ``copyOutput`` for globalFluxInterface to not copy output back to working directory. (`PR#1218 <https://github.com/terrapower/armi/pull/1218>`_, `PR#1227 <https://github.com/terrapower/armi/pull/1227>`_)
 #. `Executer` class has a ``dcType`` attribute to define the type of ``DirectoryChanger`` it will use. (`PR#1228 <https://github.com/terrapower/armi/pull/1228>`_)
+#. Removed restriction on using detailed cycle histories in equilibrium mode (`PR#1231 <https://github.com/terrapower/armi/pull/1231>`_)
```

Comment:
seems like this is pretty in the weeds and not worth documenting here. there is still a restriction that it will only use the first index...

Reply:
That restriction is in our internal plugin, and it is now documented and checked by the internal plugin.

Reply:
Yeah, the commit history exists for detail. But the release notes are supposed to be high-level changes that are easy for casual readers to understand the major changes in the release.

That has not been enforced well in the past, but that has always been the idea.  I'm trying to do better at cleaning up the notes before a release.

Reply:
Fine, it's been removed.